src/notes/notes.service.ts

```diff
@@ -332,6 +333,117 @@ export class NotesService {
     return await this.noteRepository.save(note);
   }
 
+  /**
```

Comment:
Changes in notes.service.ts probably would be better at home in permissions.service.ts, would'nt they?

Reply:
I added them here because `updateNotePersmissions` is above this.
I think the reason for this is that all functions work on Notes.
But we could decide to move them to `permissions.service.ts`. By now the permissions service is only a logic for permission checking. By moving the update functions in it would have more dependencies.

Reply:
I'd expect to find everything regarding permissions in the permissions service and would think this should be moved there…

Reply:
What do you think @davidmehren?

Reply:
I'd say it fits better into the `PermissionService`

Reply:
OK, I will move it.